src/pages/NotFound/NotFound.js

```diff
@@ -0,0 +1,46 @@
+// 404 Page
+
+import React from 'react';
+import styled from '@emotion/styled';
+import { Typography, Container } from '@mui/material';
+
+const Logo = '/logo_192x192.png';
+
+const NotFoundContainer = styled(Container)`
```

Comment:
Could you please move constants with styles to a separate file like NotFoundStyled.js so the main page would only contain presentational logic and markup?